example/basic.md

```diff
@@ -49,6 +62,62 @@ document.addEventListener('DOMContentLoaded', () => {
 });
 ```
 
+</div>
+<div class="tab-pane fade" id="legacy" role="tabpanel" aria-labelledby="legacy-tab">
+
+```HTML
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="https://storage.googleapis.com/crisp-store-locator/${package_version}/store-locator.css" />
+    <title>Store Locator Example</title>
+
+    <script type="text/javascript">
+      document.addEventListener('DOMContentLoaded', function () {
+        window["storeLocator"].createStoreLocatorMap({
+          container: document.getElementById("map-container"),
+          geoJson: 'sample.json',
```

Comment:
I was wondering if we can substitute this one with an actual URL from GCS of a sample json file (e.g. `gs://crisp-store-locator/0.5.4/sample.json`) and we can include a comment above to "Replace this with Crisp GeoJSON URL".

Reply:
This is a great idea and I got it to push these files to the bucket, but there's a CORS issue with having this json hosted somewhere different so that seems to negate any benefit to having this pointing to a real file. I did update the logo img src and add the comments though!